backend/molgenis-emx2-graphql/src/main/java/org/molgenis/emx2/json/Schema.java

```diff
@@ -25,7 +25,7 @@ public Schema(SchemaMetadata schema, boolean minimal) {
             .map(entry -> new Setting(entry.getKey(), entry.getValue()))
             .toList();
     List<TableMetadata> list = new ArrayList<>();
-    list.addAll(schema.getTablesIncludingExternal());
+    list.addAll(schema.getTables());
```

Comment:
what is the impact of this core behaviour change?

Reply:
this is the same point as previous. Should not be an issue. Tests are green so that is promising.  Just a bit of smoke testing with data explorer and forms to verify.

Reply:
this was previously reverted due to issues on certain pages ( please check pr history for notes on revert)

Reply:
ah, I will carefully review. Supposedly that should be fixed by now in the client but double check in order if there are other apps still depending on it (e.g. old catalogue components).

Reply:
I indeed found an issue in the client where it still depended on this (in getColumnIds, nested columns from refTables in other schemas couldn't be found; fix included for that too).